helix-term/src/ui/lsp.rs

```diff
@@ -48,8 +48,9 @@ impl SignatureHelp {
 }
 
 impl Component for SignatureHelp {
-    fn render(&mut self, area: Rect, surface: &mut Buffer, cx: &mut Context) {
+    fn render(&mut self, mut area: Rect, surface: &mut Buffer, cx: &mut Context) {
         let margin = Margin::horizontal(1);
+        area.height -= 1;
```

Comment:
Why the change here?

Reply:
The `clip_bottom` should probably be gated on the new config option as well - there's a noticeable blank line when the option is off

Reply:
I thought the lack of bottom margin was unintentional since other popups have margins around them, and here we only miss it from the bottom.
I changed this now, so without borders if should look as it originally was.